.github/workflows/generate_policies.yml

```diff
@@ -0,0 +1,54 @@
+name: Generate Scan Policies from Policy Tags
+on: 
+  schedule: # The start of every Friday
+    - cron: '0 0 * * 5'
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-policies:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        persist-credentials: false
+        path: zap-extensions
+        fetch-depth: 0
+    - name: Create Policies
+      run: |
+        # Run the ZAP script
+        docker run -v $(pwd):/zap/wrk/:rw --user root -t ghcr.io/zaproxy/zaproxy:nightly ./zap.sh -addoninstall ascanrulesAlpha -silent -script /zap/wrk/zap-extensions/.github/scripts/generateScanPolicies.js -cmd
```

Comment:
Is the root needed?

Reply:
I was trusting the alert pages workflow??? 🤷‍♂️

Reply:
https://github.com/zaproxy/zap-admin/blob/5866200f177712cb086738f8e0ef5cd6b1bf814c/.github/workflows/alert-docs.yml#L31

Reply:
Okay.